opennms-doc/guide-admin/src/asciidoc/text/provisioning/detectors/wsmanwql.adoc

```diff
@@ -0,0 +1,70 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+==== WS-Man WQL Detector
+
+The WS-Management WQL detector attempts to connect to the agent defined in `wsman-config.xml` and issues a WQL query.
+If the query successfully returns one or more items, the service is marked as detected. The WS-Man WQL detector can be used to define arbitrary services based on WQL filter results.
+
+===== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.wsman.WsManWQLDetector`
+|===
+
+===== Configuration and Usage
+
+.Parameters for the <DETECTOR-NAME-HERE>
```

Comment:
"<DETECTOR-NAME-HERE>"?

Reply:
Same as opennms-doc/guide-admin/src/asciidoc/text/provisioning/detectors/wsman.adoc, which was the template for the wql detector doc.  if it's wrong, it's wrong there too :)